### DISCUSSION TITLE
so-setup" Command not found" ( am starting again )

### DISCUSSION THREAD

**Discussion by wilmerism** [Security-Onion-Solutions/securityonion#9313] [General]
Apologies in advance here if im the dumb ass..Still learning,
CAnt se

Ive issues with my install and deciding to blow away ans start again.
Looking at older docs i should be able to re-run setup?

In connected via ssh
Am on version 2.3.190
Have done sudo soup all good
so-status does have 2 issues 
    so-elastalert -------------------------------------------------------------------------------------- [ MISSING ]
    so-elasticsearch ------------------------------------------------------------------------------------- [ ERROR ]

I have sound so-setup but i get command not found. Even navigating to the file have issue

[admin@emitsecon SecurityOnion]$ cd setup
[admin@emitsecon setup]$ ls
automation  install_scripts  so-functions       so-preflight  so-variables
files       public_keys      so-functions.orig  so-setup      so-whiptail
[admin@emitsecon setup]$ so-setup
-bash: so-setup: command not found
[admin@emitsecon setup]$ sudo so-setup
[sudo] password for admin:
sudo: so-setup: command not found
[admin@emitsecon setup]$

same issue with sudo bash so-setup-network

Thanks in advance

**Comment by reyesj2** [comment]
should be 
``` sudo /SecurityOnion/setup/so-setup iso```

**Reply by wilmerism** [reply]
Morning, Thanks for an answer. 

However no go >>

[admin@emitsecon ~]$ sudo /SecurityOnion/setup/so-setup -iso
[sudo] password for admin:
sudo: /SecurityOnion/setup/so-setup: command not found
[admin@emitsecon ~]$ sudo /SecurityOnion/setup/so-setup
sudo: /SecurityOnion/setup/so-setup: command not found
[admin@emitsecon ~]$

**Comment by dougburks** [comment]
Assuming you installed using our ISO image, please run `sudo SecurityOnion/setup/so-setup iso` (make sure there is no backslash before `SecurityOnion`) as shown at:
https://docs.securityonion.net/en/2.3/installation.html#installation-using-security-onion-iso-image

**Comment by RagnarSTS** [comment]
If you're sitting inside the setup folder ... then you want sudo ./so-setup -iso
dot slash forces to look for executables in current working directory.
Otherwise home directory wildcard path would be "~/SecurityOnion/setup/so-setup -iso"
or full path "/home/onion/SecurityOnion/setup/so-setup -iso"
